Vagrantfile

```diff
@@ -65,15 +63,12 @@ Vagrant.configure("2") do |config|
   # forward ssh agent to easily ssh into the different machines
   config.ssh.forward_agent = true
 
-  config.vm.box = "coreos-%s" % $update_channel
-  if $image_version != "current"
-      config.vm.box_version = $image_version
-  end
-  config.vm.box_url = "https://storage.googleapis.com/%s.release.core-os.net/amd64-usr/%s/coreos_production_vagrant_virtualbox.json" % [$update_channel, $image_version]
+  config.vm.box = "coreos-alpha"
+  config.vm.box_url = "https://storage.googleapis.com/alpha.release.core-os.net/amd64-usr/current/coreos_production_vagrant_virtualbox.json"
```

Comment:
Is there any reason not to use https://alpha.release.core-os.net/amd64-usr/current/coreos_production_vagrant_virtualbox.json instead?